src/Interface/Modules/Render/Ospray/OSPRayRenderer.cpp

```diff
@@ -56,17 +56,7 @@ OSPRayRenderer::OSPRayRenderer()
   float backgroundColor[] = {0.0, 0.0, 0.0};
   ospSetParam(renderer_, "backgroundColor", OSP_VEC3F, backgroundColor);
   ospCommit(renderer_);
-
-  float dir[] = {0.0, -1.0, -1.0};
-  float col[] = {0.0, 1.0, 1.0};
-  OSPLight light = ospNewLight("distant");
-  ospSetParam(light, "color", OSP_VEC3F, col);
-  ospSetParam(light, "direction", OSP_VEC3F, dir);
-  ospCommit(light);
-
-  //ospSetObjectAsData(world_, "light", OSP_LIGHT, light);
```

Comment:
This line being commented was the culprit. The light needed to be set for it to be used.